Neos.Neos/Classes/Domain/Service/RenderingUtility.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Neos\Neos\Domain\Service;
+
+use Neos\ContentRepository\Core\Projection\ContentGraph\Filter\FindAncestorNodesFilter;
+use Neos\ContentRepository\Core\Projection\ContentGraph\Node;
+use Neos\ContentRepositoryRegistry\ContentRepositoryRegistry;
+use Neos\Flow\Mvc\ActionRequest;
+use Neos\Fusion\Core\FusionGlobals;
+use Neos\Neos\Domain\Model\FusionRenderingStuff;
+use Neos\Neos\Domain\Model\RenderingContext;
+use Neos\Neos\Domain\Repository\SiteRepository;
+use Neos\Flow\Annotations as Flow;
+
+#[Flow\Scope('singleton')]
+class RenderingUtility
```

Comment:
Those should better be moved in the via https://github.com/neos/neos-development-collection/pull/4496 "emptied" `FusionService`